haddock-library/test/Documentation/Haddock/ParserSpec.hs

```diff
@@ -86,6 +86,9 @@ spec = do
       it "parses identifiers enclosed within backticks" $ do
         "`foo`" `shouldParseTo` DocIdentifier "foo"
 
+      it "doesn't parse a composed function as an identifier" $ do
+        "'foo.bar'" `shouldParseTo` "'foo.bar'"
```

Comment:
This looks potentially problematic: qualified identifiers should still be identified as `DocIdentifier`, no?

Reply:
But this is not a valid and qualified  haskell identifier, right?